main.go

```diff
@@ -311,6 +314,26 @@ func main() {
 		fail("Issue with input: %s", err)
 	}
 
+	//
+	// Fastlane session
+	fmt.Println()
+	log.Infof("Ensure cookies for Apple Developer Portal")
+
+	fs, err := session.GetSession()
```

Comment:
redundant and non-informative client-side naming, more info in my next comments

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/96d945ed090daf9b2988faba7e1b5ad0316997a9